src/AddTodoForm.js

```diff
@@ -1,15 +1,21 @@
-import React from "react";
+import React from 'react';
+
+const AddTodoForm = (props) => {
```

Comment:
`props` represents the entire object that's being passed from parent to child. Nice job working with it. Now, what's an advanced technique we could use to directly access each property in `props`? Not only will this give us a clear display of which props are being passed along but it'll also clean up our code when we want to access each prop.

This is a little beyond the scope of the assignment but it's a very common pattern you will encounter. You can merge if you can't solve it. Feel free to ping me for help or for another review if you are able to refactor your code.